index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <title>BEM Practice</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <section class="home">
+      <div class="header">
+        <div class="header__menu">
+          <h1 class="header__men__logo">Medical Core</h1>
+          <ul class="header__nav__list">
+            <li class="header__nav__list__item header__nav__list item--active">
+              Home
+            </li>
+            <li class="header__nav__list__item">Product</li>
+            <li class="header__nav__list__item">Pricing</li>
+            <li class="header__nav__list__item">Contact</li>
+          </ul>
+        </div>
+        <div class="header__auth">
+          <button class="btn">Sign in</button>
+          <button class="btn btn--active">
+            Join us
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              strokeWidth="{1.5}"
+              stroke="currentColor"
+              class="btn__svg"
+            >
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                d="M13.5 4.5L21 12m0 0l-7.5 7.5M21 12H3"
+              />
+            </svg>
+          </button>
+        </div>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          strokeWidth="{1.5}"
+          stroke="currentColor"
+          class="header_menu-icon"
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            d="M3.75 9h16.5m-16.5 6.75h16.5"
+          />
+        </svg>
+      </div>
+      <div class="home__presentation">
+        <div>
+          <p class="home__presentation-header color-primary">Welcome</p>
```

Comment:
is color-primary a modifier? wouldn't you have written color--primary?

Reply:
Yes it is , but I'm going to make it clear !